schemas/config.impress.definition/domains.schema

```diff
@@ -0,0 +1,11 @@
+{
+  Instance: Enum('standalone', 'controller', 'server'),
+
+  Nomens: Many({ domain: 'Nomen' }),
```

Comment:
@tshemsedinov I wonder whether I can do this, if I need to define, that the value of this domain can be an array of strings?